surefire-api/src/test/java/org/apache/maven/surefire/util/internal/StringUtilsTest.java

```diff
@@ -110,4 +110,12 @@ public void testEscapeWithHeader()
 
         assertArrayEquals( expectedResult, actualResult );
     }
+
+    public void testEmptyByteArray() {
```

Comment:
Pls put the bracket '{' on new line as we have in other main code.
Amend the previous commit means we should have single commit per fix/feature. Thx.